taskipy/pyproject.py

```diff
@@ -10,29 +10,47 @@
     MissingPyProjectFileError,
     MissingTaskipyTasksSectionError
 )
+from taskipy.variable import Variable
 
 
 class PyProject:
     def __init__(self, base_dir: Path):
         pyproject_path = self.__find_pyproject_path(base_dir)
-        self.__items = PyProject.__load_toml_file(pyproject_path)
+        self.__items = self.__load_toml_file(pyproject_path)
 
     @property
     def tasks(self) -> Dict[str, Task]:
         try:
-            return {
-                task_name: Task(task_name, task_toml_contents) for
-                task_name, task_toml_contents in self.__items['tool']['taskipy']['tasks'].items() }
+            toml_tasks = self.__items['tool']['taskipy']['tasks'].items()
         except KeyError:
             raise MissingTaskipyTasksSectionError()
 
+        tasks = {}
+        for name, toml_contents in toml_tasks:
+            tasks[name] = Task(name, toml_contents)
+
```

Comment:
Nice idea! Longer but more explicit, which helps to better understand the code :)